index.css

```diff
@@ -368,7 +376,8 @@ footer {
   padding: 0.5em;
   box-sizing: border-box;
   width: 32em;
-  color: var(--bg0);
+  color: var(--text);
+  background-color: var(--bg0);
```

Comment:
My issue here is the use of a background color (`--bg0`) as a text color.  I know this is probably a bit pedantic but my approach to themes is to have semantically named vars based on where the colors are applied (e.g. background, text, highlight).  Thus, if we need a "dim" text color we should have `--dim-text` var rather than repurposing `--bg0` (which makes it difficult to tweak bg0 later on should we feel the need.)

I'll fix this up.

Reply:
Yeah, the current theming situation is really confusing, I don't know if `--text` is gonna be dark or light. As you said, I think, there should only ever be one `.theme-*` defined globally, not per section.

Reply:
Me writing theme code:

![image](https://media0.giphy.com/media/7P2q6uXI1MRCE/giphy.gif)